rllib/contrib/az/examples/train_cartpole.py

```diff
@@ -0,0 +1,50 @@
+"""Example of using training on CartPole."""
```

Comment:
Can you include reference rewards / timesteps here so users can check if they are able to reproduce the example?